tombstone_gc.cc

```diff
@@ -285,6 +285,21 @@ std::map<sstring, sstring> get_default_tombstone_gc_mode(data_dictionary::databa
     return get_default_tombstone_gc_mode(ks.get_replication_strategy(), tm);
 }
 
+void validate_tombstone_gc_options(const tombstone_gc_options* options, const gms::feature_service& feature_service,
+        const locator::abstract_replication_strategy& rs, const locator::token_metadata& tm)
+{
+    if (!options) {
+        return;
+    }
+    if (!feature_service.tombstone_gc_options) {
+        throw exceptions::configuration_exception("tombstone_gc option not supported by the cluster");
+    }
```

Comment:
Nit (but already superseded by my previous comment): this check also happens in the other overload of `validate_tombstone_gc_options`, which calls this one. I think it would be better to remove the old one if you introduce a new one here.